src/session/session_api.c

```diff
@@ -1806,6 +1806,24 @@ __session_timestamp_transaction(WT_SESSION *wt_session, const char *config)
     API_END_RET(session, ret);
 }
 
+/*
+ * __session_timestamp_transaction_commit --
+ *     WT_SESSION->timestamp_transaction_commit method.
+ */
+static int
+__session_timestamp_transaction_commit(WT_SESSION *wt_session, unsigned long long ts)
```

Comment:
The  `wt_timestamp_t` is defined with `uint64_t`. It is better to use the same instead of `unsigned long long`?

Reply:
yes, that's fair. i was trying to be consistent with `int error` above, but after your comment i double-checked and `transaction_pinned_range` uses a `uint64_t`, so it's better to be specific about the width.

Reply:
Fixed in 97525ed